src/main/java/com/commercetools/sync/products/ProductSync.java

```diff
@@ -547,24 +552,14 @@ private CompletionStage<Void> applyCallbackAndCreate(@Nonnull final ProductDraft
         .orElse(CompletableFuture.completedFuture(null));
   }
 
-  /**
-   * Given a {@link String} {@code errorMessage}, this method calls the optional error callback
-   * specified in the {@code syncOptions} and updates the {@code statistics} instance by
-   * incrementing the total number of failed products to sync.
-   *
-   * @param errorMessage The error message describing the reason(s) of failure.
-   */
-  private void handleError(
-      @Nonnull final String errorMessage,
-      @Nullable final ProductProjection oldResource,
-      @Nullable final ProductDraft newResource) {
-    handleError(errorMessage, null, oldResource, newResource, null);
-  }
-
   /**
    * Given a {@link String} {@code errorMessage} and a {@link Throwable} {@code exception}, this
    * method calls the optional error callback specified in the {@code syncOptions} and updates the
-   * {@code statistics} instance by incrementing the total number of failed products to sync.
+   * {@code statistics} instance by incrementing the total number of failed products to sync. <br>
+   * <br>
+   * NOTE: This method is similar to {@link com.commercetools.sync.commons.BaseSync#handleError}. It
```

Comment:
Can we change the name of this method?
I don't think it is overriding from baseSync. So instead of using super.handleError() everywhere in the code. We can rename this method here and use it when required.
WDYT?

Reply:
Can you suggest the name too? 🙏

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/850/commits/6e5977279f117dfd9356016bc7cd0de613c371b2